src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -11,13 +12,25 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
+    public static final String MESSAGE_INCORRECT_FORMAT = "Invalid command format! \nFormat: clear";
     public static final String MESSAGE_SUCCESS = "ProductiveNus has been cleared!";
```

Comment:
Looks good! However, the other commands implemented the error message by using Messages.MESSAGE_INVALID_COMMAND_FORMAT ("Invalid command format!") and MESSAGE_USAGE ("Format: clear").

But this is just a minor nitpick so you can ignore this for now XD
I can go edit it to match the other classes once this PR has been merged :)